src/autopas/particles/ParticleBase.h

```diff
@@ -152,6 +157,22 @@ class ParticleBase {
    */
   void setR(const std::array<double, 3> &r) { _r = r; }
 
+  /**
+   * update rebuild position of the particle
+   */
+  void setRAtRebuild() { _rAtRebuild = _r; }
```

Comment:
```suggestion
  /**
   * Update rebuild position of the particle.
   */
  void setRAtRebuild() { _rAtRebuild = _r; }
```

Slightly controversial. This is not a setter in the typical sense because it doesn't take an argument. I'm fine with it, though.

Reply:
I made this change as I also needed an actual setter to resolve some other comment :) But still open to changing the name if necessary.